src/libs/getWindowHeightAdjustment/index.android.js

```diff
@@ -0,0 +1,4 @@
+// On Android the window height does not include the status bar height, so we need to add it manually.
+export default function getWindowHeightAdjustment(insets) {
+    return insets.top;
```

Comment:
I think it is preferable to use insets from react-native-safe-area-context instead of StatusBar.currentHeight which is a constant so it could have issues if we want to support screen rotation.

If we use insets then we do need platform files as it is implemented on all platform while StatusBar.currentHeight is android only.